stl/inc/bitset

```diff
@@ -40,6 +40,36 @@ __declspec(noalias) bool __stdcall __std_bitset_from_string_2(void* _Dest, const
 #endif // _USE_STD_VECTOR_ALGORITHMS
 
 _STD_BEGIN
+template <class _Ty, size_t _Size, bool _Dynamic = (_Size * sizeof(_Ty) > _STD_BITSET_TO_STREAM_STACK_RESERVATION)>

```

Comment:
One of the things I always say about template programming is that (especially prior to concepts) template parameters have no type system of their own, so it's especially important to rely on naming conventions to avoid surprises. In this case, it is deeply confusing (to me at least) for `_Ty` to appear here, when that has a very different meaning to `bitset` (i.e. the storage unit, either 32-bit or 64-bit). The usage is actually `_Buffer_for_bitset<_Elem, _Bits>`, so having `_Elem` "change name" is what's confusing.

Unless there's a very good reason to rename the parameters, I think we should keep the same names as the usage. This sounds like a superficial naming nitpick but it's one I really really care about.